java-frontend/src/main/java/org/sonar/plugins/java/api/CheckRegistrar.java

```diff
@@ -37,6 +38,7 @@
  * </pre>
  */
 @Beta
+@SonarLintSide
 @BatchSide
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "BatchSide"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)